qapla/app.py

```diff
@@ -48,22 +52,24 @@ def run_command(self, settings=None):
         settings = settings or {}
         self._create_app(settings, 'command')
 
-    def _create_app(self, settings={}, endpoint='uwsgi'):
-        self._generate_settings(settings, endpoint)
+    def _create_app(self, settings={}, method='uwsgi'):
+        self._generate_settings(settings, method)
         self.append_app_plugins()
+        self.is_created = True
+        self.method = method
 
     def _generate_settings(
         self,
         settings,
-        endpoint,
+        method,
     ):
         self.settings = settings
         self.paths = {}
```

Comment:
`self.paths` is assigned an empty dictionary, passed into `self.Config.settings` and overwritten at the end of this method. It seems not necessary to assign an empty dict. Maybe pass empty dict into `self.Config.settings` instead?

Reply:
Fixed, by removing of `self.settings =` and `self.paths = `